### PR TITLE
feat: add closedBy on apps-engine

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: apps/meteor/app/apps/server/converters/rooms.js [hunk 1]
```diff
@@ -210,6 +211,17 @@ export class AppRoomsConverter {
...
 				return this.orch.getConverters().get('departments').convertById(departmentId);
 			},
+			closedBy: async (room) => {
+				const { closedBy } = room;
+
+				if (!closedBy) {
+					return undefined;
+				}
+
+				delete room.servedBy;
```
Comment (KevLehman) [inline apps/meteor/app/apps/server/converters/rooms.js:221]: closedBy and servedBy can hold diff data (ex, when a manager closes a conversation that an agent is serving)

File: apps/meteor/app/apps/server/converters/rooms.js [hunk 2]
```diff
@@ -210,6 +211,17 @@ export class AppRoomsConverter {
...
+
+				return this.orch.getConverters().get('users').convertById(closedBy._id);
```
Comment (KevLehman) [inline apps/meteor/app/apps/server/converters/rooms.js:223]: The `closedBy` can also be a visitor, we need something like:
```
if (closer === 'user') return this.orch.getConverters().get('users').convertById(closedBy._id);
return return this.orch.getConverters().get('visitors').convertById(convertById(closedBy._id));

File: apps/meteor/app/apps/server/converters/rooms.js (hunk 1 above)
Comment (ggazzo) [reply apps/meteor/app/apps/server/converters/rooms.js:221]: ops

Comment (ggazzo) [inline apps/meteor/app/apps/server/converters/rooms.js:221]:
```suggestion
				delete room.closedBy;
```

**Review by KevLehman — DISMISSED** [review]

**Pull request merged**